apps/web/src/components/SearchModal/ManageTokens.tsx

```diff
@@ -94,7 +94,7 @@ export default function ManageTokens({
     <Wrapper>
       <Column style={{ width: '100%', flex: '1 1' }}>
         <AutoColumn gap="14px">
-          <Row>
+          <Flex width="96%" justifyContent="center" ml="2%">
```

Comment:
Is it possible to remove the overflow hidden on the outside container instead of tweaking the width of the input field?

Reply:
I've tried setting Row's `overflowX: visible `, but it not working🤦‍♂️

Reply:
I think should add `overflow: visible` to the `ModalBody` component.
But the current dom structure has a problem. Need to move the scroll bar from the whole modal body to the inner list first

Reply:
@chefjackson  Fixed. Thanks for your help🙏.

Reply:
> @chefjackson Fixed. Thanks for your help.🙏